clusters-rke1/README.md

```diff
@@ -1,96 +1,42 @@
-| <a name="input_cluster_name"></a> [cluster\_name](#input\_cluster\_name) | Unique identifier appended to the Rancher url subdomain | `string` | `"load-testing"` | no |
-| <a name="input_create_node_reqs"></a> [create\_node\_reqs](#input\_create\_node\_reqs) | Flag defining if a cloud credential & node template should be created on tf apply. Useful for scripting purposes | `bool` | `true` | no |
-| <a name="input_existing_node_template"></a> [existing\_node\_template](#input\_existing\_node\_template) | (Optional) Name of an existing node template to use. Only use this if create\_node\_reqs is false. | `string` | `""` | no |
-| <a name="input_iam_instance_profile"></a> [iam\_instance\_profile](#input\_iam\_instance\_profile) | n/a | `string` | `null` | no |
-| <a name="input_insecure_flag"></a> [insecure\_flag](#input\_insecure\_flag) | Flag used to determine if Rancher is using self-signed invalid certs (using a private CA) | `bool` | `false` | no |
-| <a name="input_install_docker_version"></a> [install\_docker\_version](#input\_install\_docker\_version) | The version of docker to install. Available docker versions can be found at: https://github.com/rancher/install-docker | `string` | `"20.10"` | no |
-| <a name="input_node_pool_count"></a> [node\_pool\_count](#input\_node\_pool\_count) | Number of node pools to create | `number` | `3` | no |
-| <a name="input_nodes_per_pool"></a> [nodes\_per\_pool](#input\_nodes\_per\_pool) | Number of nodes to create per node pool | `number` | `1` | no |
-| <a name="input_rancher_api_url"></a> [rancher\_api\_url](#input\_rancher\_api\_url) | api url for rancher server | `string` | n/a | yes |
-| <a name="input_rancher_token_key"></a> [rancher\_token\_key](#input\_rancher\_token\_key) | rancher server API token | `string` | n/a | yes |
-| <a name="input_security_groups"></a> [security\_groups](#input\_security\_groups) | A list of security group names (EC2-Classic) or IDs (default VPC) to associate with | `list(any)` | `[]` | no |
-| <a name="input_server_instance_type"></a> [server\_instance\_type](#input\_server\_instance\_type) | Instance type to use for rke1 server | `string` | n/a | yes |
-| <a name="input_volume_size"></a> [volume\_size](#input\_volume\_size) | Size of the storage volume to use in GB | `string` | `"32"` | no |
-| <a name="input_volume_type"></a> [volume\_type](#input\_volume\_type) | Type of storage volume to use | `string` | `"gp2"` | no |
-
-## Outputs
-
-| Name | Description |
-|------|-------------|
-| <a name="output_create_node_reqs"></a> [create\_node\_reqs](#output\_create\_node\_reqs) | n/a |
-| <a name="output_node_template"></a> [node\_template](#output\_node\_template) | n/a |
-<!-- END OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
+This terraform is for creating the downstream RKE clusters to test Ranchers scaling ability. It contains two root modules, one for AWS clusters and one for Linode clusters.
```

Comment:
Maybe there needs to be instructions for the provisioning on AWS also?

Reply:
This is a generic README that applies to both modules, the specific examples provided just happen to be for the linode module. I think its ok as-is for the current PR. The required `terraform.tfvars` for the AWS module can be mostly copied from the README with some minor changes based on the variables that exist or don't exist in the AWS module.